analyses/molecular-subtyping-HGG/07-HGG-molecular-subtyping-combine-table.Rmd

```diff
@@ -598,7 +598,10 @@ molecular_subtype_table <- all_data_df %>%
          Kids_First_Biospecimen_ID_DNA, Kids_First_Biospecimen_ID_RNA) %>%
   left_join(molecular_subtype_table) %>%
   distinct() %>%
-  replace_na(list(molecular_subtype = "HGG, H3 wildtype"))
+  dplyr::mutate(molecular_subtype = case_when(
+    is.na(molecular_subtype) & !is.na(Kids_First_Biospecimen_ID_DNA) ~ "HGG, H3 wildtype",
+    is.na(molecular_subtype) & is.na(Kids_First_Biospecimen_ID_DNA) ~ "HGG, To be classified",
+    TRUE ~ molecular_subtype))
```

Comment:
So what's left for `TRUE ~ molecular_subtype` would be whenever `!is.na(molecular_subtype)`. But if `is.na(molecular_subtype)` is NA it will be determined by whether or not there's a DNA sample. 

Just writing this out to make sure I understand what's happening and that it is what we want to happen. (I think it is).

Reply:
that's right! I'll add as comments in that section as well